armory/utils/config_loading.py

```diff
@@ -58,6 +61,11 @@ def load_model(model_config):
     )
     if not isinstance(model, Classifier):
         raise TypeError(f"{model} is not an instance of {Classifier}")
+    if weights_file is None and not model_config["fit"]:
```

Comment:
Suggestion - make this:
```
if weights_file and not model_config["fit"]:
```
Right now, if weights_file == "", it successfully runs but does not output a warning. See:
https://github.com/twosixlabs/armory/blob/master/armory/baseline_models/keras/mnist.py#L55

Reply:
Thanks. Updated to `if not weights_file and not model_config["fit"]:` is that what you meant?

Reply:
Yes, that's what I meant.